cypress/integration/seededFlows/adminFlows/users/userIndexView.spec.js

```diff
@@ -229,7 +229,7 @@ describe('User index view', () => {
 
       it(`Clicks through to the Member Detail View`, () => {
         cy.findAllByRole('link', { name: 'Admin McAdmin' }).first().click();
-        cy.url().should('contain', '/admin/users/1');
+        cy.url().should('contain', '/admin/member_manager/users/1');
```

Comment:
I'd love to use js-routes in these files to avoid having to make changes like these but I think we had an issue with it the last time we used it. Hence, I plan on adding an issue to our board and maybe during cooldown week I can look into it.